src/App.js

```diff
@@ -1,16 +1,36 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.css';
 import chatMessages from './data/messages.json';
+import ChatLog from './components/ChatLog';
 
 const App = () => {
+  const [entryData, setEntryData] = useState(chatMessages);
+
+  const [likeCount, setLikeCount] = useState(0);
+
+  const updateEntry = (entryToUpdate) => {
+    const entries = entryData.map((entry) => {
+      if (entry.id === entryToUpdate.id) {
+        return entryToUpdate;
```

Comment:
We don't actually have functionality to update an entry so we don't need to return the object passed into `updateEntry`. Since we can only like and unlike an entry, how could you refactor this method to only update likeCount?